test/integration/http2_upstream_integration_test.cc

```diff
@@ -409,7 +409,11 @@ TEST_P(Http2UpstreamIntegrationTest, UpstreamResetWithManyStreams) {
                                                                    {":scheme", "http"},
                                                                    {":authority", "host"}},
                                            *responses[i]));
-           codec_client->sendData(*encoders[i], 0, true);
+           if (i % 15 == 0) {
```

Comment:
nit: should we rename this test now to UpstreamResetOrEndStreamWithManyStreams ? (Or whatever). 
any rationale behind the % 15 here? If so maybe drop comment?